packages/block-library/src/image/view.js

```diff
@@ -187,6 +239,21 @@ const { state, actions, callbacks } = store(
 			},
 		},
 		callbacks: {
+			initImage() {
```

Comment:
This could be more explicit. Like `addImageIdToGallery` or `populateGalleryArray`.

Reply:
updated to `populateGalleryArray` in 1bf3f27a53adf605c8a556abcfedc6939eb96b54